proposals/841-release-process/README.md

```diff
@@ -0,0 +1,115 @@
+- The diverse skills required for the role make it even harder for candidates to join.
+* Release Documentation
+- Working groups need help on building Technical Documentation for Releases.
+- Working on technical documentation during the Release, before the feature freeze, can expedite the release process.
+* Feature Freeze vs Building Release Candidate
+- Teams required more time to finish the Release and later cherry-pick and work on building the Release.
+* Release timeframe 
+- The Release is now 27 weeks. There are 10 weeks between feature freeze and Release. 
+- Release dates and phases are hard to predict in the current process.
+
+### Goals
+
+* Reduce time to market of Kubeflow Release
+* Reduce meeting time and structure to give back time to the Release Team.
+* Focus on contributions from technical documents and source code by reducing meetings and promoting async communication.
+* Promote clarity on dates and phases so distributions, working groups, and the whole community can plan accordingly.
+* Improve release cycle by moving technical decisions affecting the release to the Release Management team including Kubeflow platform
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+## Proposal
+
+### Release Timelines
+KubeFlow Release x.x is moved to Quarterly Release = 12 weeks
+
+![alt text](release.png)
+
+* Notes: Release dates will not be changed unless critical changes are needed.
+Week 9 is code freeze. However, the working groups can use that week to finish any outstanding feature critical for the release.
+
+* KubeFlow Release minor versions x.x.xx
+** Release can be 60-90 days.
+
+### Release Meetings and Communication
+* Create a new Release Management team on Slack for asynchronous communication within the team
+* Bi-weekly meetings during Software Development (Week 1-Week 8). Weekly meetings between week 9-12 as needed.
+* Meetings will focus on roadmap status, including blockers, feature discussions, the overall status of the release, and the help needed.
+
+### Release Documentation
+
+* Build a team of technical writers to be embedded in each working group for the release time frame. 
+* Provide roles within the community to boost participation and bring new members into this group.
+* Technical Doc Lead will lead this team; technical documentation can include a blog and slides to announce the Release.
+
+### Release Management Team: 
+The Release Management Team is composed by: Release Manager, WG Leads ( in some cases Liasons), Technical Doc Lead.
+Responsabilities:
+
+- WG leads from each component (ex., Pipelines) must approve a synchronized PR on the Kubeflow Platform/Manifests when related to that component 
+- WG leads from each component (ex., Pipelines) must approve a PR when updates regarding each component on the Kubeflow Platform/Manifests.
```

Comment:
```suggestion
```

It should be an automatic continuous process. We rely on automatic testing, not manual approvals.

Reply:
And they are. There are github notifications and this is how we did it over the last years.